pkg/util/namespace.go

```diff
@@ -26,6 +26,10 @@ func IsNamespaceExist(client kubeclient.Interface, namespace string) (bool, erro
 
 // CreateNamespace just try to create the namespace.
 func CreateNamespace(client kubeclient.Interface, namespaceObj *corev1.Namespace) (*corev1.Namespace, error) {
+	// It's necessary to set the label of namespace to make sure that the namespace is created by Karmada.
+	namespaceObj.Labels = map[string]string{
+		ManagedByKarmadaLabel: ManagedByKarmadaLabelValue,
+	}
```

Comment:
it is a public function and should only be responsible for creating the namespace. I don't tend to add the label here, preferably at the function call.

Reply:
make sense.

Reply:
+1